src/js/mixins/AppActionsHandlerMixin.jsx

```diff
@@ -197,9 +197,11 @@ var AppActionsHandlerMixin = {
       if (instancesString != null && instancesString !== "") {
         let instances = parseInt(instancesString, 10);
 
-        if (model.container != null && model.container.volumes != null &&
-          model.instances > instances
-        ) {
+        if (model.container != null &&
```

Comment:
Please also fix the [`handleSuspendApp`](https://github.com/mesosphere/marathon-ui/blob/fix/stateful-warning/src/js/mixins/AppActionsHandlerMixin.jsx#L279) condition, and maybe move the persistent volume app condition into a util.